uint8 task_index   # 任务索引
run_task/ObjPositionWithID[] obj_positions # 物体位置信息